.travis.yml

```diff
@@ -14,7 +15,7 @@ cache:
 
 env:
   matrix:
-    - SYMFONY_VERSION=2.8.* 
+    - SYMFONY_VERSION=3.1.*
   global:
     - SYMFONY_DEPRECATIONS_HELPER=weak
```

Comment:
missing `- SYMFONY_PHPUNIT_REMOVE="symfony/yaml" SYMFONY_PHPUNIT_DIR=".phpunit"` and then also adding `.phpunit` to the cache directories.